src/components/chat/components/send-box/SendBox.styles.ts

```diff
@@ -0,0 +1,57 @@
+import { styled } from 'styled-components';
+
+import { Theme } from '../../../../shared/theme.types';
+import { Icon } from '../../../icon';
+
+export const StyledSendBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 25px;
+`;
+
+export const StyleInputGroup = styled.div<{ $error: boolean }>`
```

Comment:
Typo:
```suggestion
export const StyledInputGroup = styled.div<{ $error: boolean }>`
```

Reply:
💭 I was wondering if it's worth reusing the shared styles for inputs like so:
```javascript
export const StyleInputGroup = styled.div<InputStylesProps>`
  ${inputBorderStyle}
  ${inputBackgroundStyle}
  ${inputBackgroundStyle}
  ${inputCursorStyle}
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 12px;
  border-radius: 12px;
```

Reply:
We should export `InputStylesProps` to use it here
```suggestion
export const StyleInputGroup = styled.div<InputStylesProps>`
```